flow/cmd/peer_data.go

```diff
@@ -348,6 +349,39 @@ func (h *FlowRequestHandler) GetSlotInfo(
 	}, nil
 }
 
+func (h *FlowRequestHandler) GetSlotLag(
+	ctx context.Context,
+	req *protos.GetSlotLagRequest,
+) (*protos.GetSlotLagResponse, error) {
+	rows, err := h.pool.Query(ctx, `
+    select updated_at, slot_size;
+    from peerdb_stats.peer_slot_size
+    where slot_size is not null
+      and slot_name = $1
+      and updated_at > (now()-$2::INTERVAL)
+    order by random()
```

Comment:
why?

Reply:
sampling so that 720 points are randomly distributed as opposed to being clustered when looking at past month